packages/koa-shopify-auth/src/auth/test/index.test.ts

```diff
@@ -0,0 +1,158 @@
+import {createMockContext} from '@shopify/jest-koa-mocks';
+
+import createShopifyAuth from '../index';
+
+import createTopLevelRedirect from '../create-top-level-redirect';
+jest.mock('../create-top-level-redirect');
```

Comment:
This mock doesn't seem to work and I can't figure out why. It's the cause of the test failures. Any ideas?

Reply:
What does 'doesn't seem to work' mean? Have you tried manually mocking like you do in the other test?

Reply:
Within some of the tests I do something like:

```
const mockTopLevelRedirect = jest.fn();
createTopLevelRedirect.mockImplementation(mockTopLevelRedirect);
```

You can see the resultant test failures: when my code attempts to call the function returned by `createTopLevelRedirect` (which should be `jest.fn()` in this case) I get `[function name] is not a function`.

Reply:
ohhh, try doing `require.requireMock('../create-top-level-redirect')` instead of the import in your test file.